contrib/bash-completion

```diff
@@ -23,9 +23,17 @@ _netctl() {
         COMPREPLY=( $(compgen -W "--help --version list store restore stop-all start stop restart switch-to is-active status enable disable reenable is-enabled edit wait-online" -- "$cur") )
       ;;
       2)
-        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(start|stop|restart|switch-to|is-active|status|enable|disable|reenable|is-enabled|edit|wait-online) ]] &&
+        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(is-active|status|enable|disable|reenable|is-enabled|edit|wait-online) ]] &&
           compopt -o filenames &&
           mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(_netctl_profiles)" -- "$cur")
+        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(stop|restart) ]] &&
+          compopt -o filenames &&
+	  # Suggest only active profiles for stopping/restarting
+          mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(netctl list | grep '\*' | cut -f2 -d' ')" -- "$cur")
+        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(switch-to|start) ]] &&
+	  # Suggest only inactive profiles for starting/switching-to
+          compopt -o filenames &&
+          mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(netctl list | grep -v '\*' | sed 's/^[\t ]*//g')" -- "$cur")
```

Comment:
In line with the previous `sed` proposal, this could become `sed -n 's/^  //p'`.

Reply:
Or, for maximum POSIX:
```suggestion
          mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(netctl list | sed -nE 's/^[[:space:]]+//p')" -- "$cur")
```

Reply:
Why would we complicate things like this? We control the output of `netctl list` and it is a simple ` `. Let's not pretend we are more complex than we are! Although GitHub does not show it, there are two spaces between `s/^` and `//p` in my previous comment.